test/integration/serverless/recorder-extension/go.mod

```diff
@@ -0,0 +1,41 @@
+module datadog-lambda-extension/recorder-extension
+
+go 1.14
```

Comment:
Go 1.14 is no longer supported, ideally you would use Go 1.16 which is what we have installed in our builders